### PR TITLE
Add style and testing workflows

### PR DESCRIPTION
**Pull request by xjules**
In order to test new workflows, they are called from python-app.yml. Once successful I'll remove python-app.yml and remove the workflow_call trigger.

### CODE REVIEW

File: .github/workflows/style.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+          pip install black ruff
+          pip install .
+
+      - name: Run black
+        run: |
+          black --check .
+
+      - name: Run ruff
+        run: |
+          ruff komodoenv
```
Comment (berland) [inline .github/workflows/style.yml:31]: Ruff the tests also?

File: .github/workflows/testing.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+  workflow_call:
+
+jobs:
+  tests:
+    name: "Python ${{ matrix.python-version }}"
+    runs-on: ubuntu-20.04
```
Comment (berland) [inline .github/workflows/testing.yml:13]: ubuntu-latest?

Comment (pinkwah) [reply .github/workflows/testing.yml:13]: Ubuntu 20.04 is the last Ubuntu to have Python 3.8 available in `/usr`.

**Review by berland — APPROVED** [review]
:+1:

**Pull request merged**